policyengine/__init__.py

```diff
@@ -1 +1 @@
-VERSION = "1.70.1"
+VERSION = "1.70.2"
```

Comment:
Just a heads up, we don't actually need to update the version number here. Instead, could you [update the changelog](https://github.com/PolicyEngine/policyengine/blob/main/changelog.yaml)? We have a GitHub action that automatically updates this file, CHANGELOG.md and package.json. See an example update [here](https://github.com/PolicyEngine/policyengine/pull/808/files).